handbook/engineering/2021_org.md

```diff
@@ -1,46 +1,38 @@
-# 2021 Q1 Organizational Plan
+# 2021 Engineering Organization Plan
 
 Defining a future vision for the organization is necessary to effectively grow the engineering team. It is understood that all plans change, and this document will be kept up to date with these changes. This is our written plan to align the team and clarify hiring and growth expectations.
 
-- VP Engineering
-  - Director of Engineering: Core Products
-    - [Search Team](search/index.md)
-      - Engineering Manager: Search
-      - Product Manager: Search
-      - Backend Engineers: 3
-      - Frontend Engineers: 2
-    - [Campaigns Team](campaigns/index.md)
-      - Engineering Manager: Campaigns
-      - Product Manager: Campaigns
-      - Backend Engineers: 3
-      - Frontend Engineers: 2
-    - [Code Intelligence Team](code-intelligence/index.md)
-      - Engineering Manager: Code Intel
-      - Product Manager: Code Intel
-      - Backend Engineers: 3
-      - Frontend Engineers: 2
-    - [Extensibility Team](web/index.md)
-      - Engineering Manager: Extensions
-      - Product Manager: Extensions
-      - Frontend Engineers: 4
-    - [Code Host Integrations Team](web/index.md)
-      - Engineering Manager: Code Host Integrations
-      - Product Manager: Code Host Integrations
-      - Frontend Engineers: 4
-  - Director of Engineering: Core Services
-    - [Backend Infrastructure Team](backend-infrastructure/index.md)
-      - Engineering Manager: Backend Infrastructure
-      - Backend Engineers: 4
-      - Dev Ops Engineers: 2
-    - [Distribution Team](distribution/index.md)
-      - Engineering Manager: Distribution
-      - Product Manager: Distribution
-      - Dev Ops Engineers: 4
-      - Backend Engineers: 2
-    - [Security Team](security/index.md)
-      - Engineering Manager: Security
-      - Backend Engineers: 3
-      - Security Engineers: 3
-    - [Web Infrastructure Team](web/index.md)
-      - Engineering Manager: Web Infrastructure
-      - Frontend Engineers: 4
+[2021 headcount plan](https://docs.google.com/spreadsheets/d/1UG4aLB5S8yOSOByia2EQgpZkoPxG5eVfqQqgs1f71Vw/edit#gid=2097845084)
+
+## Product engineering team
+
+This organization is responsible for building products that deliver value to our customers and drive business growth. Each product engineering team has the autonomy and the necessary cross-functional skills (for example: frontend engineers, backend engineers, site reliability engineers, product manager, designer) to achieve their mission.
```

Comment:
@nicksnyder do you intend for the site reliability engineering role to fall under the span of influence of the engineering manager of the product team? Or will they be a counterpart to the team like the PM and Designer is and report to a different manager?

Reply:
Good question. My default instinct is to follow general principles (teams should have full stack ownership) and say necessary SREs will be embedded on the team and report to the EM of the product team. If our monitoring/alerting infra is good enough (built by operations platform team), then it might be unnecessary to have specialized SREs on product teams (frontend/backend engineers can manage the alerts for those areas of the product).